components/io-libs/adios2/SPECS/adios2.spec

```diff
@@ -0,0 +1,159 @@
+elements, their types, and how you wish to process them this run, the
+routines in the host code (either Fortran or C) can transparently change
+how they process the data.
+
+%prep
+%setup -q -n %{PNAME}-%{version}
+
+%build
+mkdir adios2-build
+cd adios2-build
+
+# OpenHPC compiler/mpi designation
+%ohpc_setup_compiler
+
+module load phdf5
+module load %{python_module_prefix}numpy
+module load %{python_module_prefix}mpi4py
+
+export CC=mpicc
+export CXX=mpicxx
+export F77=mpif77
+export FC=mpif90
+export MPICC=mpicc
+export MPIFC=mpif90
+export MPICXX=mpicxx
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=%{install_path} \
+    -DBUILD_SHARED_LIBS=ON \
+    -DADIOS2_USE_MPI=ON \
+    -DADIOS2_USE_HDF5=ON -DHDF5_ROOT="${HDF5_DIR}" \
+    -DADIOS2_USE_ZeroMQ=OFF \
+    -DADIOS2_USE_Fortran=ON \
+    -DADIOS2_USE_Python=ON \
+    -DADIOS2_USE_SST=ON \
+    -DADIOS2_USE_BZip2=ON \
+    -DBUILD_TESTING=ON \
+    ..
+make
+# make test
+
+
+%install
+# OpenHPC compiler designation
+%ohpc_setup_compiler
+
+make DESTDIR=$RPM_BUILD_ROOT install
+
+# this is clearly generated someway and shouldn't be static
+export PPATH="/lib64/%{python_lib_dir}/site-packages"
+export PATH=$(pwd):$PATH
+
+
+# OpenHPC module file
+%{__mkdir} -p %{buildroot}%{OHPC_MODULEDEPS}/%{compiler_family}-%{mpi_family}/%{pname}
+%{__cat} << EOF > %{buildroot}/%{OHPC_MODULEDEPS}/%{compiler_family}-%{mpi_family}/%{pname}/%{version}
+#%Module1.0#####################################################################
+
+proc ModulesHelp { } {
+
+puts stderr " "
+puts stderr "This module loads the %{PNAME} library built with the %{compiler_family} compiler"
+puts stderr "toolchain and the %{mpi_family} MPI stack."
+puts stderr "\nVersion %{version}\n"
+
+}
+module-whatis "Name: %{PNAME} built with %{compiler_family} compiler and %{mpi_family} MPI"
+module-whatis "Version: %{version}"
+module-whatis "Category: runtime library"
+module-whatis "Description: %{summary}"
+module-whatis "%{url}"
+
+set             version             %{version}
+
+depends-on phdf5
+depends-on %{python_module_prefix}numpy
+depends-on %{python_module_prefix}mpi4py
+
+prepend-path    PATH                %{install_path}/bin
+prepend-path    INCLUDE             %{install_path}/include
+prepend-path    LD_LIBRARY_PATH     %{install_path}/lib64
+
+setenv          %{PNAME}_DIR        %{install_path}
+setenv          %{PNAME}_BIN        %{install_path}/bin
+setenv          %{PNAME}_LIB        %{install_path}/lib64
+setenv          %{PNAME}_INC        %{install_path}/include
+
+EOF
+
+%{__cat} << EOF > %{buildroot}/%{OHPC_MODULEDEPS}/%{compiler_family}-%{mpi_family}/%{pname}/.version.%{version}
+#%Module1.0#####################################################################
+##
+## version file for %{pname}-%{version}
+##
+set     ModulesVersion      "%{version}"
+EOF
+
+%{__mkdir_p} ${RPM_BUILD_ROOT}/%{_docdir}
+
+%files
+%{OHPC_HOME}
+%doc Copyright.txt
+%doc Contributing.md
+%doc LICENSE
+%doc ReadMe.md
```

Comment:
This is the content of `adios2-build` folder after successful `make`:

```
$ ls -1F
adios2-config.cmake
adios2-config-common.cmake
adios2-config-install.cmake
adios2-config-version.cmake
adios2-c-targets.cmake
adios2-cxx11-targets.cmake
adios2-fortran-targets.cmake
adios2_reorganize_wrapper*
adios2-targets.cmake
bin/
bindings/
bpls.cmake.gen
cmake/
CMakeCache.txt
CMakeFiles/
cmake_install.cmake
compile_commands.json
CTestCustom.cmake
CTestTestfile.cmake
DartConfiguration.tcl
examples/
install_manifest.txt
lib/
lib64/
Makefile
plugins/
Release/
source/
testing/
Testing/
thirdparty/
verinfo.h
```

Do I need to add more entries to `%files` ?